tutorials/resnet.html

```diff
@@ -0,0 +1,77 @@
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>TVM RPC Test Page</title>
```

Comment:
is it possible to move it to tutorials/web?

Reply:
Moved.